docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -500,6 +500,19 @@ Step 12: Start a Kubernetes StatefulSet for MongoDB
     backend port. Set it to the value specified for ``mongodb-backend-port``
     in the ConfigMap.
 
+  * **Optional**: You can also change the value for ``STORAGE_ENGINE_CACHE_SIZE``, for more information
+    regarding this configuration, please consult the `MongoDB Official
+    Documentation <https://docs.mongodb.com/manual/reference/configuration-options/#storage.wiredTiger.engineConfig.cacheSizeGB>`_.
+
+  * **Optional**: If you are not using the **Standard_D2_v2** virtual machines for Kubernetes agents as per the guide,
+    please update the ``resources`` for ``mongo-ss``. We suggest allocating ``memory`` using the following scheme
+    for a MongoDB StatefulSet:
+
+    .. code:: bash
+
+      memory = (Total_Memory_Agent_VM_GB - 2GB)
+      STORAGE_ENGINE_CACHE_SIZE = memory / 2
```

Comment:
I would suggest that we link this from our doc to MongoDB docs, so that in case they change it tomorrow, we are not left with outdated info here.

Reply:
This is not from an official MongoDB `docs`. It is a from a use case/presentation on MongoDB website.